src/lib/rest/StringFilter.cpp

```diff
@@ -1307,7 +1671,18 @@ bool StringFilter::match(ContextElementResponse* cerP)
       break;
 
     case SfopMatchPattern:
-      return itemP->matchPattern(caP);
+      //
+      // To not be forgotten for CNR (and discussed during PR-review):
```

Comment:
Yeah. I added the NOTE in the function header, to try to minimize the risk of this happening again.
Probably nobody will read it ... :-D